pages/charcuterie.js

```diff
@@ -1,24 +1,31 @@
-import Head from 'next/head'
-import Carousel from '../components/Carousel'
-import Footer from '../components/Footer'
+import Head from 'next/head';
+import Carousel from '../components/Carousel';
+import Footer from '../components/Footer';
+import { ContentContainer } from '../components/ContentContainer';
 
 export default function Charcuterie() {
   return (
     <>
       <Head>
         <title>Charcuterie | nwPlus</title>
       </Head>
-
       <main>
-        <h1>charcuterie 😋</h1>
-        <p>Just a place where we put things</p>
-
-        <h2>Footer</h2>
+        <ContentContainer>
```

Comment:
Shall we make the `Wrapper` in `ContentContainer` a `main` element? ie
```
const Wrapper = styled.main`
```

Reply:
Good suggestion @ianmah @kevin-zou! but when I made the wrapper a `main` element and wrapped it around all the elements in the charcuterie, it looks like the colored backgrounds got pushed inwards as well (since `Wrapper` applies padding to its children).
![Screen Shot 2021-08-13 at 11 38 23 AM](https://user-images.githubusercontent.com/38872354/129407965-613d7dbf-7398-4b45-825f-456821e039cc.jpg)

**To resolve this,** I was thinking that **instead of having the wrapper (or `ContentContainer`) wrap the entire page, we can make it wrap individual elements on the page** (i.e. we would use `ContentContainer` in the internals of each page instead of externally wrapping all the pages).

Does this make sense?

Reply:
sounds good! Yeah forgot that the coloured backgrounds would be messedup

Reply:
Ah I see, yeah that totally makes sense to use it for each section 👍 Thanks for the explanation!